src/main/java/io/github/dftrakesh/cloverrest/model/Order/ListOrderLineItems.java

```diff
@@ -0,0 +1,11 @@
+package io.github.dftrakesh.cloverrest.model.Order;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import lombok.Data;
+import java.util.List;
+
+@Data
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class ListOrderLineItems {
+    private List<OrderLineItems> elements;
```

Comment:
change name elements to orderLineItemList

Reply:
changed the elements name to orderLineItemList